packages/salesforcedx-utils-vscode/src/telemetry/telemetry.ts

```diff
@@ -70,13 +73,30 @@ export class TelemetryService {
     return TelemetryService.instance;
   }
 
+  /**
+   * Initialize Telemetry Service during extension activation.
+   * @param context extension context
+   * @param extensionName extension name
+   */
   public async initializeService(
     context: ExtensionContext,
-    machineId: string
+    extensionName: string
   ): Promise<void> {
     this.context = context;
-    this.cliAllowsTelemetry = await this.checkCliTelemetry();
+    this.extensionName = extensionName;
+
+    this.checkCliTelemetry()
+      .then(async cliEnabled => {
+        this.setCliTelemetryEnabled(
+          this.isTelemetryExtensionConfigurationEnabled() && cliEnabled
+        );
+      })
+      .catch();
```

Comment:
We should log (console.log) a message in case there is an exception here.

Reply:
Good suggestion - Added in https://github.com/forcedotcom/salesforcedx-vscode/pull/2628/commits/9cb62bb99c8beb056d43c66785555640556fe3e1.